test/test_pubsub.py

```diff
@@ -3,8 +3,15 @@
 # Copyright (C) 2022 Collabora Limited
 # Author: Michal Galka <michal.galka@collabora.com>
 
+# Copyright (C) 2022 Collabora Limited
+# Author: Alexandra Pereira <alexandra.pereira@collabora.com>
+
```

Comment:
```suggestion
# Author: Alexandra Pereira <alexandra.pereira@collabora.com>
```

There is no need to add another `Copyright` header. Just add yourself as an author.